Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.5.2 (unreleased)
 
 Language Features:
+ * Do not warn about unused variables or state mutability for functions with an empty body.
```

Comment:
Is "not warning about something anymore" really a language feature?

Reply:
Moved to `Compiler Features`